geowebcache/core/src/test/java/org/geowebcache/util/FileMatchers.java

```diff
@@ -159,7 +163,7 @@ public void describeMismatch(Object item, Description description) {
             @Override
             public boolean matches(Object item) {
                 if(item instanceof File) {
-                    return ((File) item).isDirectory() && filesMatcher.matches(((File) item).listFiles());
+                    return ((File) item).isDirectory() && filesMatcher.matches(Arrays.asList(((File) item).listFiles()));
```

Comment:
Did this just not work before?
Seems like the sort of change that deserves its own unit test.

Reply:
No, it appears this was broken but wasn't being used.